logfire/_internal/integrations/httpx.py

```diff
@@ -226,11 +195,11 @@ def content(self) -> bytes:
             raise ValueError('Cannot read content from a streaming body')
         return list(self.stream)[0]  # type: ignore
 
-    @property
+    @cached_property
     def text(self) -> str:
-        return decode_body(self.content, self.content_type_charset)
+        return self.content.decode(self.content_type_charset)
```

Comment:
Please inline this property, it's simple enough that it's not useful to users, the fact that it can raise errors can surprise users, and it doesn't actually need to be cached.